### PR TITLE
Cart and wishlist page

### PR DESCRIPTION
**Pull request by omkarpatke**
added cart and wishlist page as well as cart items and wishlist items

### CODE REVIEW

**Comment by prankurpandeyy** [comment]
looks good, the last button over poster is overflowing please fix .

**Review by prankurpandeyy — APPROVED** [review]

File: components/cart/cart.css [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    flex-direction: column;
+    background-color: var(--gray-bg-color);
+}
+
+.cart-container{
+    display: flex;
+    padding: 2rem 2rem;
```
Comment (yaswanthmyneni) [inline components/cart/cart.css]: you can remove one 2rem.

File: components/cart/cart.css [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+}
+
+.product{
+    display: flex;
+    line-height: 1.7;
+    padding: 1rem;
+    margin: 1rem 1rem;
+    width: auto;
+    height: auto;
+    border-top: 1px solid #DDD;
```
Comment (yaswanthmyneni) [inline components/cart/cart.css]: use css variables for colors.

File: components/cart/cart.css (hunk 1 above)
Comment (omkarpatke) [reply components/cart/cart.css]: oh ok. thanks for the suggestion.

File: components/wishlist/wishlist.css [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+    padding: 2rem 3rem;
+}
+
+.wishlist-container{
+    background-color: var(--white-color);
+    padding: 2rem;
+}
+
+.wishlist-heading{
+    border-bottom: 2px solid #DDD;
```
Comment (yaswanthmyneni) [inline components/wishlist/wishlist.css:17]: use css variables.

File: style.css [hunk 4]
```diff
@@ -0,0 +1,265 @@
...
+
+/* sub-header */
+
+.sub-header{
+    display: flex;
+    justify-content: flex-end;
+    background-color: var(--primary-color);
+    width: 100%;
+    position: fixed;
+    z-index: 3;
```
Comment (yaswanthmyneni) [inline style.css:27]: use CSS variables for z-index

File: style.css [hunk 5]
```diff
@@ -0,0 +1,265 @@
...
+    z-index: 1;
+    font-size: 12px;
+}
+
+
+
+/* home page */
+
+.main-section{
+    padding: 0 4rem 0 4rem;
```
Comment (yaswanthmyneni) [inline style.css:151]: you can remove last 0 4rem,

**Review by yaswanthmyneni — COMMENTED** [review]
you can merge after making changes. your pages look good.

**Pull request merged**